misc/helm-charts/publish.sh

```diff
@@ -171,8 +171,26 @@ else
   run_if_not_dry git push origin main "$TERRAFORM_HELM_VERSION"
   cd ..
 
+  echo "--- Bumping terraform-aws-materialize"
+  rm -rf terraform-aws-materialize
+  git clone https://github.com/MaterializeInc/terraform-aws-materialize.git
+  cd terraform-aws-materialize
+  # Update environmentd version in materialize-instance module
+  sed -i "s/\".*\"\(.*\) # META: mz version/\"$CI_MZ_VERSION\"\\1 # META: mz version/" modules/materialize-instance/variables.tf
+  # Update operator version in operator module
+  sed -i "s/\".*\"\(.*\) # META: helm-chart version/\"$CI_HELM_CHART_VERSION\"\\1 # META: helm-chart version/" modules/operator/variables.tf
+  # Generate docs using the custom script
+  .github/scripts/generate-docs.sh
+  git config user.email "noreply@materialize.com"
+  git config user.name "Buildkite"
+  git add modules/materialize-instance/variables.tf modules/operator/variables.tf README.md
+  git commit -m "Bump to operator $CI_HELM_CHART_VERSION, materialize $CI_MZ_VERSION"
+  git diff HEAD~
+  run_if_not_dry git push origin main
```

Comment:
This is just a copy of how we currently bump the `terraform-helm-materialize` mdouel, open to suggestions if we should do this the same way for the new refactored AWS module.

We could leave the helm chart version as null in the Terraform module so it would default to the latest helm version [here](https://github.com/MaterializeInc/materialize/blob/gh-pages/index.yaml), but we would still need the bump for the envd image version.